pkg/imgpkg/cmd/push_test.go

```diff
@@ -235,6 +239,86 @@ func TestImageAndBundleLockError(t *testing.T) {
 	}
 }
 
+func TestLabels(t *testing.T) {
```

Comment:
The only other test that I think we can do is when you provide a multi work label value like: `my.label="something that has multiple words"`